### ISSUE TITLE
Photos problem

### ISSUE THREAD

**Issue by aymanSalahA** [conorc470/django-ecommerce-project#14]
They are not loading in your app on Heroku

**Comment by temirovazat** [comment]
Yes, heroku has the property of not storing images in itself, and for this it is better to use third-party platforms for storing images in the form of Amazon s3
